### PR TITLE
Add a generatedTime value to JSON generated by the API.

### LINKED ISSUE #596: Add timestamp to each API response
Provide a date/timestamp to each API response to offer contextual information on the age of a result, e.g. *the RPKI validty was x based on a validation run done at moment y*.

### PR DESCRIPTION
**Pull request by partim**
Fixes #596

### CODE REVIEW

**Comment by AlexanderBand** [comment]
The man page still says "The VRP list is placed into a JSON object with a single element _roas_ which contains an array of objects with four elements each...". It should mention the _metadata_ element and the objects.

**Comment by AlexanderBand** [comment]
Using the `validate` subcommand in JSON format generates broken output, e.g.

```
$ routinator validate --json --asn 12654 --prefix 93.175.147.0/24

{
  "validated_routes": [
    {
      "route": {
        "origin_asn": "AS12654",
        "prefix": "93.175.147.0/24"
      },
      "validity": {
        "state": "invalid",
        "reason": "as",
        "description": "At least one VRP Covers the Route Prefix, but no VRP ASN matches the route origin ASN",
        "VRPs": {
          "matched": [
          ],
          "unmatched_as": [
            {
              "asn": "AS196615",
              "prefix": "93.175.147.0/24",
              "max_length": "24"
            }
          ],
          "unmatched_length": [
          ]
        }
      }
    }
  ],,
  "generatedTime": "2021-07-21T08:18:02Z"
}
```

Also, I guess you'd want a `metadata` element here as well.

**Comment by partim** [comment]
> Also, I guess you'd want a metadata element here as well, and generated.

Not really. I only did that for the /json thing because there is an established format. There certainly is no reason to add `generated`.

**Review by AlexanderBand — CHANGES_REQUESTED** [review]
The man page item I'm referring to is [here](https://github.com/NLnetLabs/routinator/blob/0631562c25c4fd18087c02a6eca6626910dc8147/doc/routinator.1#L488)

File: doc/routinator.1 [hunk 1]
```diff
@@ -502,10 +502,19 @@ except for different naming of the trust anchor. Routinator uses the name
 of the TAL file without the extension
 .IR ".tal"
 whereas the RIPE NCC Validator has a dedicated name for each.
+.IP
+The output object also includs a member named
```
Comment (AlexanderBand) [inline doc/routinator.1:506]: s/includs/includes

File: doc/routinator.1 [hunk 2]
```diff
@@ -541,6 +550,15 @@ will provide the path of the local exceptions file and, optionally,
 .I "comment"
 will provide the comment if given for the assertion.
 .IP
+The output object also includs a member named
```
Comment (AlexanderBand) [inline doc/routinator.1:553]: s/includs/includes

**Review by AlexanderBand — CHANGES_REQUESTED** [review]
typos

**Review by AlexanderBand — APPROVED** [review]

**Review by DRiKE — APPROVED** [review]

**Pull request merged**